src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -31,7 +31,9 @@ def import_from_package
       end
 
       def show
+        @image.build_description if @image.description.nil? # Because the form needs a Description object instantiated
```

Comment:
Isn't better... and is more self explanatory, that way maybe you can remove the comment
```ruby
@image.build_description unless @image.description.present?
```

Reply:
I'm pretty sure that rubocop will tell you to use `if` over `unless` here. Also the comment explains why this is necessary, and changing the code like you suggested still doesn't explain that.